### DISCUSSION TITLE
New Models Folder not Detected by Voyager

### DISCUSSION THREAD

**Discussion by pmreyes** [thedevdojo/voyager#5436] [Q&A] [answered]
Hi,

I've tried to find if this question has been asked before but I had no luck finding the right answer.
We've upgraded to Laravel 8 and moved all Models in App/ to App/Models/

I updated the below codes to reference the correct namespace for our Models folder in the voyager.php config file:
```
'user' => [
        'add_default_role_on_register' => true,
        'default_role'                 => 'user',
        'admin_permission'             => 'browse_admin',
        'namespace'                    => App\Models\User::class,
        'redirect'                     => '/admin',
    ],
```
```
    'models' => [
        'namespace' => 'App\\Models\\',
    ],

```

I've also tried to execute the php artisan commands to clear the config cache but each time I view the Users table for example, I always get the below error:

`
Target class [App\User] does not exist. (View: /var/www/html/resources/views/vendor/voyager/bread/browse.blade.php)
`

I'm getting the same error on some of my models

I'd like to ask if there is something that I'm missing to update so it would look at App\Models\User rather than App\User. It seems that it's not reading the voyager config file correctly.

Thanks in advance for your time!

**Comment by emptynick** [comment, marked as answer]
The model is stored in your database as well.
Check the BREAD builder and set the model there as well.

**Reply by pmreyes** [reply]
Thanks for your time @emptynick ! That did the job! :+1:
